### ISSUE TITLE
Normalization method are different in pretrained model and transfer_learning_minc

### ISSUE THREAD

**Issue by chinakook** [dmlc/gluon-cv#576]
In the Resnet50_v2 [training script](https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/classification/imagenet/resnet50_v2.sh), the net is normalized with ```mean_rgb=[123.68, 116.779, 103.939]``` and ```std_rgb = [58.393, 57.12, 57.375]```. However, in the [transfer_learning_minc tutorial](https://gluon-cv.mxnet.io/build/examples_classification/transfer_learning_minc.html), the pretrained Resnet50_v2 is normalized with ```transforms.ToTensor()``` and then ```mean=[0.485, 0.456, 0.406]```, ```std=[0.229, 0.224, 0.225]```. It's mismatched.

**Comment by zhreshold** [comment]
I dont' think we are still using `mean_rgb=[123.68, 116.779, 103.939] and std_rgb = [58.393, 57.12, 57.375]` anywhere in GluonCV now. These are the legacy values for legacy models.

**Comment by chinakook** [comment]
If ```--use-rec``` is taken, the legacy ```mx.io.ImageRecordIter``` will be used. As we can see, most training scripts of GluonCV does take ```--use-rec```.

**Comment by zhreshold** [comment]
@hetong007 if you have some comments

**Comment by hetong007** [comment]
With record io we normalize the image before the pixels are mapped into [0, 1], while with data loader we normalize afterwards. These vectors are conceptually equivalent. Although the computation may introduce minor numerical difference, I think is not significant.

**Comment by hetong007** [comment]
@chinakook sorry I made a wrong statement. 

> With record io we normalize the image before the pixels are mapped into [0, 1]

Actually we don't map pixels into [0, 1] with recordio. We directly normalize them with `mean_rgb=[123.68, 116.779, 103.939]` and `std_rgb = [58.393, 57.12, 57.375]`, given that the input pixels are in [0, 255]. After the normalization there's no need to deal with the scale anymore.

This is *almost* equivalent to 1) map [0, 255] -> [0, 1] and 2) normalize pixels with `mean=[0.485, 0.456, 0.406]` and `std=[0.229, 0.224, 0.225]`

**Comment by chinakook** [comment]
I've got it.It's mathematically equalities.Thanks.
